src/main/scala/outwatch/SideEffects.scala

```diff
@@ -1,10 +0,0 @@
-package outwatch
-
-import monix.execution.Ack.Continue
-import monix.execution.Scheduler
-
-trait SideEffects {
```

Comment:
Why did you remove the sideEffects helpers?

Reply:
I couldn't get them to compile so I deleted them for now, and then forgot to readd, as I wasn't done